src/main/java/frc/robot/Constants.java

```diff
@@ -225,6 +267,7 @@ public static class LimelightConstants {
     public static final double FIELD_HALF_WIDTH = 3.837865;
     public static final double TAG_Y_POS = 1.451102;
     public static final double TAG_HEIGHT = 0.4572;
+    public static final double APRILTAG_STALE_TIME = 1;
```

Comment:
Is there a reason to use a double here? Also, having the unit in the name might be useful.

Reply:
to allow for higher precision tuning